### ISSUE TITLE
Since 2.4.0 do not support .NETFramework 4.5.2

### ISSUE THREAD

**Issue by smorzhov** [commandlineparser/commandline#407]
Trying to install the latest version via Nuget got the following error. However, version 2.3.0 was installed successfully.
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not install package 'CommandLineParser 2.4.0'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.5.2', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.			0	
```

**Comment by wischi-chr** [comment]
That is also true for .NET 4.0 which should be compatible according to the readme - it that still the case?

Edit: Maybe this issue should be closed, looks like a duplicate of #388

**Comment by ericnewton76** [comment]
Also a duplicate of #387

We have a fix coming.

**Comment by ooples** [comment]
I'm also needing a fix for this issue because I use .net 4.5.2

**Comment by moh-hassan** [comment]
This issue is resolved in the release [2.5.0](https://www.nuget.org/packages/CommandLineParser/2.5.0)
